recipes/tarts/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "tarts" %}
+{% set version = "0.0.0a4" %}  # Update this to your current version
+{% set python_min = "3.8" %}
```

Comment:
We typically ask that `python_min` is not explicitly set unless it is greater than the current conda-forge global pin (currently 3.9).